quickfixj-base/src/main/java/quickfix/MessageFactory.java

```diff
@@ -53,9 +53,9 @@ default Message create(String beginString, ApplVerID applVerID, String msgType)
      * Creates a group for the specified parent message type and
      * for the fields with the corresponding field ID
      *
-     * Example: to create a {@link quickfix.fix42.MarketDataRequest.NoMDEntryTypes}
+     * Example: to create a {quickfix.fix42.MarketDataRequest.NoMDEntryTypes}
      * you need to call
```

Comment:
Same here but with `@link`

Reply:
The @link tag causes Javadoc warnings or errors, as the base does not depend on the message jars. I did not want to be aggressive and lose the information that the author intended so I just took away the "@link" so that no errors result.